cypress.json

```diff
@@ -2,14 +2,14 @@
   "baseUrl": "http://localhost:8000",
   "viewportWidth": 1201,
   "viewportHeight": 900,
-  "defaultCommandTimeout": 5000, 
+  "defaultCommandTimeout": 15000, 
   "fixturesFolder": "test/cypress/fixtures",
   "integrationFolder": "test/cypress/integration",
   "pluginsFile": "test/cypress/plugins",
   "screenshotsFolder": "test/cypress/screenshots",
   "supportFile": "test/cypress/support",
   "videosFolder": "test/cypress/videos",
-  "video": true,
+  "video": false,
```

Comment:
Can you revert these two changes? I see no reason for extending the timeout (unless you're aware of one?), and we definitely want the video output for the Cypress dashboard. It helps with debugging.

Reply:
Oops, I forgot to revert these custom changes before committing. The extended timeout was apparently necessary on my end to pass some tests, and while video is a powerful debugging tool, it takes a very long time for me so I only enable it as a last resort